scipy/signal/ltisys.py

```diff
@@ -445,6 +423,13 @@ def impulse(self, X0=None, T=None, N=None):
         """
         return impulse(self, X0=X0, T=T, N=N)
 
+    def impulse2(self, X0=None, T=None, N=None, **kwargs):
```

Comment:
Yep, though I figured since currently both functions exist it makes sense to have them as part of the class as well. Once impulse2/step2 get removed they can be removed here as well.